lib/agoneum/web/templates/layout/app.html.slim

```diff
@@ -0,0 +1,23 @@
+doctype html
```

Comment:
Should this be `.slim` or `.slime`? the [Vim Slime syntax plugin](https://github.com/slime-lang/vim-slime-syntax/blob/master/ftdetect/slime.vim) shows `.slime`